docs/api-docs/storefront/scripts-overview.md

```diff
@@ -147,7 +147,6 @@ Scripts can not be injected to:
 - giftcertificates.php
 - sitemap.php
 
-
 ## Related resources
```

Comment:
Did you add Related resources to "On this page"?

Reply:
Yes